include/mbedtls/x509_crt.h

```diff
@@ -77,6 +77,9 @@ typedef struct mbedtls_x509_crt
     mbedtls_x509_buf subject_id;        /**< Optional X.509 v2/v3 subject unique identifier. */
     mbedtls_x509_buf v3_ext;            /**< Optional X.509 v3 extensions.  */
     mbedtls_x509_sequence subject_alt_names;    /**< Optional list of Subject Alternative Names (Only dNSName supported). */
+    mbedtls_x509_name hardware_module_name;     /**< Optional OtherName subject Alternative Name. */
```

Comment:
Note: The storage of names and sequences in `mbedtls_x509_name` and `mbedtls_x509_sequence` is very RAM heavy, and I am removing them in #2478, which will also affect these new fields.